src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_RENT = 7;
+  const LONG_RENT_DISCOUNT = 50;
+  const SHORT_RENT = 3;
+  const SHORT_RENT_DISCOUNT = 20;
+
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_RENT) {
+    return basePrice - LONG_RENT_DISCOUNT;
```

Comment:
The LONG_RENT_DISCOUNT should be subtracted from the basePrice only when the long rent duration is exactly equal to 7. If the rent duration is more than 7, the discount should be more.

Reply:
https://mate.academy/chats/27190